XenAdmin/Wizards/PatchingWizard/PatchingWizard_SelectServers.cs

```diff
@@ -222,6 +236,14 @@ private void EnabledRow(Host host, UpdateType type, int index)
                     return;
                 }
 
+                
+                if (poolOfOne != null && PoolForbidsAutomaticUpdates(poolOfOne))

```

Comment:
For performance reasons I would move this check to the beginning, I don'st see why we should start calculating upgrading sequences if the pool does not support automatic updates.